sale_commission_product_criteria_domain/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# © 2023 ooops404
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+{
+    "name": "Sale Commission Product Criteria Domain",
+    "version": "14.0.1.0.0",
+    "author": "Ilyas," "Ooops404," "Odoo Community Association (OCA)",
+    "contributors": ["Ilyas"],
+    "maintainers": ["ilyasProgrammer"],
+    "website": "https://github.com/OCA/commission",
+    "category": "Sales Management",
+    "license": "AGPL-3",
+    "depends": [
+        "sale_commission_product_criteria_discount",
```

Comment:
This dependency shows a non-desired coupling between modules. Maybe this should be split into 2: one for base product criteria, and another for the discounts.

Reply:
Thats right. I redid it to depend only on sale_commission_product_criteria